src/psbt.h

```diff
@@ -174,49 +159,83 @@ struct PSBTInput
             SerializeToVector(s, final_script_witness.stack);
         }
 
+        // Issuance value
+        // We shouldn't have both the value and value commitment, but maybe we do
```

Comment:
The comments should make the code more clear, rather than introduce doubt.

"but maybe we do" is not connected to the action in the code, and should either be dropped, or expanded, "but maybe we do, and in this case we ignore the value"

Reply:
Clarified